### PR TITLE
bump tiffimages version

### PR DESCRIPTION
**Pull request by tlnagy**

### CODE REVIEW

**Comment by tlnagy** [comment]
@sjkelly do you mind merging this when you get a chance? The CI failure appears unrelated.

**Comment by sjkelly** [comment]
Hmm it looks like CI didn't run at all though...

**Comment by stemann** [comment]
#68 <strike>should fix CI (except for `Invalidations`)</strike> fixed CI

File: Project.toml [hunk 1]
```diff
@@ -27,7 +27,7 @@ OpenEXR = "0.3"
 PNGFiles = "0.3, 0.4"
 QOI = "1"
 Sixel = "0.1.2"
-TiffImages = "0.3, 0.4, 0.5, 0.6, 0.7, 0.8"
+TiffImages = "0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9"
```
Comment (asinghvi17) [inline Project.toml]:
```suggestion
TiffImages = "0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 0.10"

File: Project.toml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 name = "ImageIO"
 uuid = "82e4d734-157c-48bb-816b-45c225c6df19"
 authors = ["Ian Butterworth"]
-version = "0.6.7"
```
Comment (stemann) [inline Project.toml:4]: Not strictly related to this PR, but the repo is missing a tag/release for 0.6.7 it seems...

Comment (IanButterworth) [reply Project.toml:4]: Fixed, but TagBot isn't happy even though I updated it to latest config and refreshed the DOCUMENTER_KEY etc. so I had to make a manual release after the tag was made

**Review by IanButterworth — APPROVED** [review]
Going off tests passing, and TiffImages 0.10 being used in tests, this LGTM.

However, we claim to support a lot of older TiffImages versions. Maybe that's true but it highlights the general need to test lower compat bounds too

**Pull request merged**